misk-hibernate/src/main/kotlin/misk/jdbc/PingDatabaseService.kt

```diff
@@ -47,7 +47,7 @@ class PingDatabaseService @Inject constructor(
     dataSource.setLoginTimeout((config.connection_timeout.toMillis() / 1000).toInt())
     dataSource.connect().use { c ->
       check(c.createStatement().use { s ->
-        s.executeQuery("SELECT 1 FROM dual").uniqueInt()
+        s.executeQuery("SELECT 1").uniqueInt()
```

Comment:
The "dual" virtual table is not supported by PG/Cockroach, but this should work on MySQL/Vitess and Postgres.